### PR TITLE
add hibernate

### PR DESCRIPTION
**Pull request by soaptherope**

### CODE REVIEW

**Comment by soaptherope** [comment]
will rewrite tests and ci tomorrow)

**Comment by soaptherope** [comment]
okay, all good =)

File: src/main/java/org/andersen/model/Apartment.java [hunk 1]
```diff
@@ -1,24 +1,36 @@
 package org.andersen.model;
 
+import jakarta.persistence.*;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.io.Serializable;
 
 @Data
```
Comment (Alexmansar) [inline src/main/java/org/andersen/model/Apartment.java]: https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/
please, read the article. don't use Data with Enitties))

File: src/main/java/org/andersen/service/ApartmentService.java [hunk 2]
```diff
@@ -3,19 +3,24 @@
...
-
-    Optional<Apartment> findApartmentById(int id);
-
     List<Apartment> sortByStatus(int pageNumber, int pageSize);
 
     List<Apartment> sortById(int pageNumber, int pageSize);
 
     List<Apartment> sortByPrice(int pageNumber, int pageSize);
 
     List<Apartment> sortByNameOfClient(int pageNumber, int pageSize);
```
Comment (Alexmansar) [inline src/main/java/org/andersen/service/ApartmentService.java]: maybe we can use it in the separate interface, like ApartmentSortService
and here you just new fields?

or, create 1 basic interface, but generic))
same for repo

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@soaptherope 
iN GENERAL GOOD JOB) jUST FEW COMMENTS

File: src/main/java/org/andersen/model/Apartment.java (hunk 1 above)
Comment (soaptherope) [reply src/main/java/org/andersen/model/Apartment.java]: got it, we only need to compare primary keys and see that they are not null

**Pull request merged**